buildroot/bin/build_example

```diff
@@ -5,13 +5,19 @@ if [ "$1" != "internal" ]; then
   exit 1
 fi
 
-SED=$(which gsed || which sed)
+SED=$(which gsed 2>/dev/null|| which sed)
 HERE=`dirname "$0"`
 
+
+if [ -z $SED ]; then
+  echo "No sed found, please install sed"
+  exit 1
+fi
+
 echo "Testing $3:"
 
 shopt -s nullglob
-for sub in find $2/config/examples/$3 -type d; do
+for sub in $2/config/examples/$3/*/; do
```

Comment:
This fails to get any subfolders deeper than one level. Many configurations are at a deeper location.

Reply:
Then using ** will be better. find is not available on windows, and the syntax is different on MacOS.

Reply:
Ah, right! Gotta remember to remember that about `bash` `for`.

Reply:
Darn. Seems to work fine in `for` under `zsh` but under `bash` only the top level appears.